### PR TITLE
use mesos 0.23 in tests

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by tnachen** [comment]
LGTM

**Comment by vieux** [comment]
I don't understand why, but I think it's related to the CI running overlayfs

File: test/integration/helpers.bash [hunk 1]
```diff
@@ -98,7 +98,6 @@ function swarm_manage() {
 	local port=$(($SWARM_BASE_PORT + $i))
 	local host=127.0.0.1:$port
 
-	echo "$SWARM_BINARY" -l debug manage -H "$host" --heartbeat=1s $discovery
```
Comment (aluzzardi) [inline test/integration/helpers.bash]: My bad :)

File: test/integration/mesos/mesos_helpers.bash [hunk 2]
```diff
@@ -27,8 +27,12 @@ function start_mesos() {
 # Start the swarm manager in background.
 function swarm_manage_mesos() {
 	local current=${#DOCKER_CONTAINERS[@]}
-	"$SWARM_BINARY" -l debug manage -H "$SWARM_HOST" --cluster-driver mesos-experimental --cluster-opt mesos.user=daemon 127.0.0.1:$MESOS_MASTER_PORT &
-	SWARM_PID=$!
+	local i=${#SWARM_MANAGE_PID[@]}
```
Comment (aluzzardi) [inline test/integration/mesos/mesos_helpers.bash]: Does this function still need to exist?

**Comment by chanwit** [comment]
I am trying to do regression on an AUFS box but Mesos test cases seem to be failing somehow.

File: test/integration/mesos/mesos_helpers.bash (hunk 2 above)
Comment (vieux) [inline test/integration/mesos/mesos_helpers.bash]: what do you mean ? the use if `$i` ?

**Comment by vieux** [comment]
@chanwit my bad, I wasn't related to overlay, but the patch was bad, I hardcoded some ips related to my env. Should be good now.

**Comment by chanwit** [comment]
@vieux it passed now :):):)
LGTM

**Comment by chanwit** [comment]
ping @aluzzardi

**Comment by vieux** [comment]
ping @aluzzardi I removed the function

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**